Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Profile.hs

```diff
@@ -64,12 +67,21 @@ data UpdateProfileReq = UpdateProfileReq
     language :: Maybe Maps.Language,
     gender :: Maybe Person.Gender,
     bundleVersion :: Maybe Version,
-    clientVersion :: Maybe Version
+    clientVersion :: Maybe Version,
+    disability :: Maybe DisabilityItem,
+    hasDisability :: Maybe Bool
   }
   deriving (Generic, ToJSON, FromJSON, ToSchema)
 
 type UpdateProfileResp = APISuccess.APISuccess
 
+data DisabilityItem = DisabilityItem
+  { id :: Id DisabilityItem,
```

Comment:
Expecting user to enter Id in update request is not correct.

Reply:
Id will come from list disability api